Eastern-Research-Group/mywaterway
# Feature/831 update dependencies

## Related Issues:
* [HMW-831](https://jira.epa.gov/browse/HMW-831)
* [HMW-832](https://jira.epa.gov/browse/HMW-832)

## Main Changes:
* Updated dependencies, it's been a couple weeks since the last update.
* Updated the changelog.

## Steps To Test:
1. Install updated dependencies
2. Test various features
3. (Optional) Run coverage and verify all tests passk, assuming the WQP services are stable. If you do this step, be sure to delete the `CYPRESS-TEST-HMW` items from your ArcGIS Online account.
    * Note: I ran the coverage tests and they all passed except for 2 of them. The 2 that failed was because the Water Quality Portal services have been up and down today. I ran the 2 that failed later and they passed.
4. Read new changelog entries and verify all items have been captured.
